esmvaltool/cmor/check.py

```diff
@@ -427,6 +428,22 @@ def _check_time_coord(self):
                 self.report_error(msg, var_name, self.frequency)
                 break
 
+    CALENDARS = [
+        ['gregorian', 'standard'],
+        ['proleptic_gregorian'],
+        ['noleap', '365_day'],
```

Comment:
I believe you also need to switch the 'noleap' and 'all_leap' calendars: https://github.com/SciTools/cf-units/blob/master/cf_units/__init__.py